website/src/blogs/2017-09-11-not-yet-another-distributed-storage-system.md

```diff
@@ -0,0 +1,55 @@
+
+Arguably, if you need hundreds of gigabytes of throughput for a particular workload, the Linux kernel comes preloaded with [one](http://www.orangefs.org/). So you have to look no further from a tech support side of things — as what better experts to find that concern themselves with the Linux kernel and thus OrangeFS, right? No need to search any further, right?
+
+Let’s first try to understand why a lot of people I’ve spoken to in the past typically chose **“scale-out.”** It seems they all like the idea of “**add another box”** to add performance and capacity. The fact that you can’t scale performance decoupled from capacity is what they take for granted. However, due to economics forcing IT segments to do things cheaper and more predictably, this is not the case — anymore.
+
+Some storage vendors have found a solution for this, by not selling you expensive boxes but rather just the software. You simply scale in any cloud on any hardware or so they say. However, this **“any any”** approach does not fit in with the requirement to make things more predictable in fact quite the opposite.
+
+So if we summarize the downsides:
+
+- Distributed storage difficult to develop and is hard and nasty to manage in production
+- Specialized drivers needed to unleash the real potential
+- Scaling the number of nodes does not decrease latency, in fact, usually the opposite (depending on implementation)
+- Best scales bandwidth and IOPS, however, this is not what a typical workload requires
+- Complex consistency models create surprises regarding what is on disk or not
+- Not especially good at leveraging flash or NVMe to deliver great performance and a small footprint
+- Big blast radius — the more data you can put in one system, the more you might lose or at least lose access to when you most need it
+- The complexity and need for quick metadata updates across nodes both argue against multi-cloud deployments (though there are some scale-out file systems being built that claim to address these issues)
+
+Now there will always be vendors out there that claim to have solved it all, unlimited scale-out, never ending IOPS, more bandwidth than the whole of the internet combined, and latency — sure we do that too…
+
+At [OpenEBS](https://www.openebs.io/), however, we took a different approach by not trying to solve the distributed problems but to take a step back and try to determine what are the real problems people need to solve?
+
+Speaking to our early tech-preview customers, we were shocked with awe to see that due to the complexity of storage these days, they simply revert to [Direct Attached Storage](https://en.wikipedia.org/wiki/Direct-attached_storage) (DAS). “It won’t go any quicker than that”, they say. And you have to make a very, very, strong case to argue against that. In fact, come to think of it, it is nearly impossible if you consider the speed of NMVe devices.
```

Comment:
```suggestion
Speaking to our early tech-preview customers, we were shocked with awe to see that due to the complexity of storage these days, they simply revert to [Direct Attached Storage](https://en.wikipedia.org/wiki/Direct-attached_storage) (DAS). `It won’t go any quicker than that`, they say. And you have to make a very, very, strong case to argue against that. In fact, come to think of it, it is nearly impossible if you consider the speed of NMVe devices.
```

Reply:
Text is added in quotations. Since this is not code, I think ` is not required